docs/quickstart.rst

```diff
@@ -19,22 +19,22 @@ First, create an `APISpec <apispec.APISpec>` object, passing basic information a
         )
     )
 
-Add definitions to your spec using `definition <apispec.APISpec.definition>`.
+Add schemas to your spec using `schema <apispec.core.Components.schema>`.
```

Comment:
The link to `schema` isn't working here because `Components` isn't being autodoc'd.

We're currently generating autodoc for `apispec`, not `apispec.core`. So you can fix this by import `Components` in `apispec/__init__.py` and changing this to

```suggestion
Add schemas to your spec using `schema <apispec.Components.schema>`.
```

Reply:
I didn't like the idea of exposing `Components` in __init__.py just for documentation sake, so I added it to the API docs through core.py.

I also removed duplicate `BasePlugin` documentation.